lib/features/registration/pages/signup_page.dart

```diff
@@ -243,7 +250,9 @@ class _SignUpPageState extends State<SignUpPage> {
               style:
                   Theme.of(context).textTheme.bodyLarge?.copyWith(fontSize: 16),
               decoration: InputDecoration(
-                hintText: AppLocalizations.of(context).lastName,
+                hintText: isUS
```

Comment:
Is surname/lastname not the same thing? Do we need to make a difference here based on country?

Reply:
surename is UK english, lastname is US. In the slack thread we discussed changes should be based on selected country not on phone language.
https://nfcollect.slack.com/archives/C04PS1D7ARL/p1703596032962579

Reply:
If lastname is US english we should change this based on phone language (Users that have US as language on their phone always want to see that, even if they use Givt in an other country). 

For the changes that are showing givt4kids specific things its indeed better to do it based on country :) as we only want to have that flow if its a US Givt user.

Reply:
okay, the term is "surname" and will be surname in UK& default english, but "Last name" if the phone language is set to US eng.